init.lua

```diff
@@ -3,13 +3,19 @@ local MODPATH = minetest.get_modpath("wiki")
 
 wikilib = { }
 
+-- TODO: understand if this can be avoided
+-- https://github.com/minetest-mods/wiki/issues/5
 local ie = minetest.request_insecure_environment()
 assert(ie, "you must allow `wiki` in `secure.trusted_mods`")
 
 local private = { }
 
 private.open = ie.io.open
-private.mkdir = ie.core.mkdir
+
+-- Note: it's not possible anymore to call ie.core.mkdir
```

Comment:
This comment is unnecessary

Reply:
Do you need that I remove all these 3 inline comments?

To be honest it may be useful since it's not clear to me why `ie.core.mkdir` it's not available anymore.

Reply:
removed in https://github.com/minetest/minetest/pull/11950. Use `minetest.mkdir` directly. It contains the necessary path access checks. If this is not enough, open an issue in minetest/minetest about this problem.